deploy/docker_push.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -ex
+
+# Install the awscli tool and add it to PATH
+pip install --user awscli
+export PATH=$PATH:$HOME/.local/bin
+
+# Authenticate with AWS and push the docker images so that they can be deployed to the server
+eval $(aws ecr get-login --no-include-email --region ap-southeast-2)
```

Comment:
`set +x` before this to avoid printing out the docker key

`set -x` afterwards to return to normal